eng/packaging.targets

```diff
@@ -19,9 +19,8 @@
                                        '$(IsRIDSpecificProject)' == 'true') and
                                        '$(PreReleaseVersionLabel)' != 'servicing' and
                                        '$(GitHubRepositoryName)' != 'runtimelab'">true</GeneratePackageOnBuild>
-    <GeneratePackageOnBuild Condition="'$(GeneratePackageOnBuild)' != 'true'">false</GeneratePackageOnBuild>
     <!-- When in source-build we need to generate all packages when building for all configurations even in servicing. -->
-    <GeneratePackageOnBuild Condition="!$(GeneratePackageOnBuild) and
+    <GeneratePackageOnBuild Condition="'$(GeneratePackageOnBuild)' != 'true' and
```

Comment:
Those conditions were "cleaned-up" in an unrelated change. This change just uses a different style for the condition: instead of `!$(GeneratePackageOnBuild)` it now uses `'$(GeneratePackageOnBuild)' != 'true'` which is the syntax that I prefer. Mainly because it also works when the property isn't set.